paralytics/preprocessing/featureunion.py

```diff
@@ -0,0 +1,109 @@
+import numpy as np
+import pandas as pd
+from sklearn.externals.joblib import Parallel, delayed
+from sklearn.pipeline import FeatureUnion, _fit_transform_one, _transform_one
+from scipy import sparse
+
+"""
+Based on
+https://github.com/marrrcin/pandas-feature-union
+
+The main goal of this transformer it to return pandas DataFrame object using 
+FeatureUnion capabilities from scikit-learn.
+"""
+
+class PandasFeatureUnion(FeatureUnion):
+    def fit_transform(
+                      self,
+                      X: pd.DataFrame,
```

Comment:
Great to see the type hinting but for now the recommended way is to write the types inside the docstring to keep consistency.